webui/src/pages/repositories/repository/compare.jsx

```diff
@@ -197,6 +206,20 @@ const CompareList = ({ repo, reference, compareReference, prefix, onSelectRef, o
                         withCommits={true}
                         withWorkspace={false}
                         selectRef={onSelectCompare}/>
+
+                    <OverlayTrigger placement="bottom" overlay={
+                        <Tooltip>Switch directions</Tooltip>
+                    }>
+                    <span>
+                        <Button variant={"link"} disabled={false}
+                              onClick={e => {
```

Comment:
Sorry to sound like a broken record here, but handlers shouldn't be anonymous arrow functions and should use `useCallback` to keep the callback function reference stable between renders

Reply:
Thanks, I learned something new :)